File: Node.java
package elvis.practice.linkedlist;

public class Node {
	
	public Object data;
	public Node next;
	
	public Node(Object data) {
		this.data = data;
		this.next = null;
	}
	
	public Node(Object data, Node next) {
		this.data = data;
		this.next = null;
	}
	
//	public Object getData() {
//		return this.data;
//	}
//	
//	public Node getNext() {
//		return this.next;
//	}
//	
//	public void setNext(Node node) {
//		this.next = node;
//	}
//	
//	public Object setData(Object data) {
//		return this.data = data;
//	}

	@Override
	public String toString() {
		return "Node [data=" + data + ", next=" + next + "]";
	}
	
	
}

File: LinkedList/LinkedList_Python/node.py
class Node:
	def __init__(self, data, next = None):
		self.data = data
		self.next = next
	
	def __str__(self) -> str:
		# return f"  Node Data: {self.data}\nNext Node: {self.next}"
		return "{ Node { data: " + self.data.__str__() + ", next: " + (self.next.__str__() if self.next is not None else "null") + " }"